packages/fluentui/e2e/tests/menuDismissOnItemClick-test.ts

```diff
@@ -0,0 +1,51 @@
+import { menuItemClassName } from '@fluentui/react-northstar';
+
+const menuItem = `.${menuItemClassName}`;
+
+const firstSubmenuItem = '#first';
+const secondSubmenuItem = '#second';
```

Comment:
maybe declare these in the example and import them so there is only one place to modify these ids ?

Reply:
Done